src/pages/Defi/components/OpportunityCard.tsx

```diff
@@ -50,6 +52,9 @@ export const OpportunityCard = ({
   const asset = useAppSelector(state => selectAssetById(state, assetId))
   const { cosmosStaking } = useModal()
   const isCosmosStaking = chainId === cosmosChainId
+  const icon = useMemo(() => {
```

Comment:
please do not put asset specific logic in this component, we need to see a generic solution that is extensible in future

the guidance i gave to a previous contributor was as follows

make a function that takes the assetId, and returns an override asset icon url

don't hardcode the URLs - get them from the asset service

use a object of the original asset id, to the override asset id, then get the icon from the override asset

if there isn't an override, just use the original icon